pages/agent/v3/hooks.md

```diff
@@ -1,102 +1,111 @@
 # Buildkite Agent hooks
 
-Hooks extend or override the built-in behaviours of the Buildkite agent and
-Bootstrap binaries.
+Hooks allow you extend or override the built-in behaviour of the Buildkite Agent as it runs jobs in a variety of phases using a variety of programming languages.
+
+### What's a hook?
+
+At its core, a hook is a script that is executed or sourced by the Buildkite agent at a specific point in the job lifecycle. Hooks can be used to extend or override the built-in behaviour of the Buildkite agent. Hooks are generally run as shell scripts, which the agent then executes or sources.
```

Comment:
English editor suggestion from not-an-English-editor:

“At its core, ” can probably be deleted.

> Hooks are generally run as shell scripts, which the agent then executes or sources.

That somewhat implies they're run and then executed 😁
Maybe “Hooks are generally shell scripts, …”?